packages/server-wallet/src/chain-service/__chain-test__/chain-service.test.ts

```diff
@@ -59,6 +65,24 @@ describe('fundChannel', () => {
   });
 });
 
+it('Fund erc20', async () => {
+  const channelId = randomChannelId();
+  const tokenContract: Contract = new Contract(
+    erc20Address,
+    ContractArtifacts.TokenArtifact.abi,
+    ethWallet
+  );
+  await (await tokenContract.increaseAllowance(erc20AssetHolderAddress, BN.from(5))).wait();
```

Comment:
Should increasing the allowance be part of the `fundChannel` method?

Reply:
Good question. Probably yes, at least in the short term. There might be use cases where a wallet wants to approve many tokens for use in multiple channels for an asset holder.

Reply:
It's not uncommon to approve an infinite amount, and in our case it may be reasonable considering the contract would be the ERC20 asset holder which has safe code that we know can't be manipulated by anyone except the caller.